implementation/visualisation/src/main/java/org/inaetics/dronessimulator/visualisation/Game.java

```diff
@@ -36,7 +37,9 @@ public Game() {
     public void start(Stage primaryStage) {
 
         if(this.subscriber == null) {
-            this.subscriber = new RabbitSubscriber(new ConnectionFactory(), "visualisation", new JavaSerializer());
+            ConnectionFactory connectionFactory = new ConnectionFactory();
+//            connectionFactory.setHost("rabbitmq");
```

Comment:
The visualisation does not run inside a docker container and can therefore connect with the rabbitmq container using localhost. Attempting to connect with "rabbitmq" would fail, unless you would configure this is your machine's hosts file.